### PR TITLE
Mudkip voting bug fix and 23 players bug fix

### PR DESCRIPTION
**Pull request by Xinverse**
- Fixed a bug where multi lynching and abstaining in mudkip breaks the game loop
- Fixed a bug where games cannot run with more than 22 players

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -2983,12 +2983,20 @@ async def reply(message, text, cleanmessage=True):
         text = text.replace('@', '@\u200b')
     await client.send_message(message.channel, message.author.mention + ', ' + str(text))
 
+async def send_long_post(channel, post):
+    if len(post) <= 2000:
```
Comment (belguawhale) [inline bot.py]: nit: Please put `2000` in a constant called something like `DISCORD_MAX_MESSAGE_LENGTH`

File: bot.py [hunk 2]
```diff
@@ -3029,6 +3037,21 @@ async def parse_command(commandname, message, parameters):
         else:
             await log(2, 'User ' + message.author.name + ' (' + message.author.id + ') tried to use command ' + commandname + ' with parameters `' + parameters + '` without permissions!')
 
+async def send_long_log_helper(channel, post, depth):
```
Comment (belguawhale) [inline bot.py]: nit: We can use a default argument value here (`depth=0`) so we don't have to specify it when calling this function

**Review by belguawhale — COMMENTED** [review]
Nice work, some minor comments.

File: bot.py (hunk 1 above)
Comment (Xinverse) [reply bot.py]: Done!

File: bot.py (hunk 2 above)
Comment (Xinverse) [reply bot.py]: Done as well!

File: bot.py [hunk 3]
```diff
@@ -3050,7 +3050,7 @@ async def send_long_log_helper(channel, post, depth):
             await client.send_message(channel, "[CONTINUED] " + "```py\n" + post[:max] + "```")
         else:
             await client.send_message(channel, post[:max] + "```")
-        await send_long_log_helper(channel, post[max:], depth + 1)
```
Comment (belguawhale) [inline bot.py]: You still need the `depth + 1` here to keep track of the number of continues; you can remove the `0` from https://github.com/belguawhale/Discord-Werewolf/pull/53/files#diff-f289368d1ae6d2d58bec9f799832d932R3070 though

Comment (Xinverse) [reply bot.py]: Thanks for catching this mistake! Fixed it in the new commit.

**Review by belguawhale — APPROVED** [review]
LGTM!

**Pull request merged**